cf_remote/commands.py

```diff
@@ -800,13 +800,15 @@ def deploy(hubs, masterfiles):
     ):
         masterfiles = "out/masterfiles.tgz"
         print("Found cfbs policy set: '{}'".format(masterfiles))
-    elif masterfiles.startswith(("http://", "https://")):
+    elif masterfiles and masterfiles.startswith(("http://", "https://")):
         urls = [masterfiles]
         paths = _download_urls(urls)
         assert len(paths) == 1
         masterfiles = paths[0]
         log.debug("Deploying downloaded: %s" % masterfiles)
     else:
+        if not masterfiles:
+            masterfiles = "."
```

Comment:
I would add a print message similar to line 802 for this. Also, maybe we can perform some sanity check first? Maybe look for `promises.cf`?

Reply:
ok, here's what it does now:

```
Craigs-MacBook-Pro.local ? cf-remote deploy --hub vagrant@ubuntu-20
cf-remote: No cfbs or masterfiles policy set found
Craigs-MacBook-Pro.local ? cd cfe/play-cfbs/
Craigs-MacBook-Pro.local ? cf-remote deploy --hub vagrant@ubuntu-20
Found cfbs policy set: 'out/masterfiles.tgz'
Craigs-MacBook-Pro.local ? cd ../masterfiles/
Craigs-MacBook-Pro.local ? cf-remote deploy --hub vagrant@ubuntu-20
Found masterfiles policy set: '/Users/craig/cfe/masterfiles'
```